src/scripts/main.js

```diff
@@ -1 +1,108 @@
 'use strict';
+
+let isClickedLeft = false;
+let isClickedRight = false;
+let timer;
+const notification = document.createElement('div');
+
+notification.setAttribute('data-qa', 'notification');
+
+function resetTimer() {
+  clearTimeout(timer);
+
+  timer = setTimeout(() => {
+    if (!isClickedLeft) {
+      firstPromise();
+    }
+  }, 3000);
+}
+
+function handleFirstClick() {
+  isClickedLeft = true;
+  firstPromise();
+  resetTimer();
+
+  document.removeEventListener('click', handleFirstClick);
+}
+
+document.addEventListener('click', handleFirstClick);
+
+document.addEventListener('mousedown', (e) => {
+  if (e.button === 0) {
+    isClickedLeft = true;
+  }
+
+  if (e.button === 2) {
+    isClickedRight = true;
+  }
```

Comment:
There is a potential issue here. The `mousedown` event is triggered before the `click` event. This means that the `isClickedLeft` variable will be set to `true` before the `click` event listener has a chance to run, which could cause unexpected behavior. Consider revising the order of these events or using a different event.